spec/factories/users.rb

```diff
@@ -0,0 +1,24 @@
+include RandomData
+
+FactoryGirl.define do
+  pw = RandomData.random_sentence
+  factory :user do
+    name RandomData.random_name
+    sequence(:email){|n| "user_#{n}@factory.com" }
+    password_digest pw
+    username RandomData.random_sentence
```

Comment:
I think it was just habit. I thought about using faker, but only after I had set up RandomData.